frontend/src/components/user/registerPage.vue

```diff
@@ -1,9 +1,207 @@
 <template>
-  <div>Rejestracja</div>
+  <div class="register-container">
+    <h1 class="title">Rejestracja</h1>
+  
+    <br>
+
+    <form @submit.prevent="submitForm">
+      <b-field label="Imię">
+          <b-input v-model="name" 
+              maxlength="30"
+              required>
+          </b-input>
+      </b-field>
+
+      <b-field label="Nazwisko">
+          <b-input v-model="surname" 
+              maxlength="30"
+              required>
+          </b-input>
+      </b-field>
+
+      <b-field label="Email">
+          <b-input type="email" v-model="email"
+              maxlength="30"
+              required>
+          </b-input>
+      </b-field>
+
+      <b-field label="Login">
+          <b-input v-model="login"
+              maxlength="30" 
+              required>
+          </b-input>
+      </b-field>
+
+      <b-field label="Hasło">
+          <b-input type="password" v-model="password"
+              password-reveal
+              minlength="8"
+              maxlength="30"
+              required>
+          </b-input>
+      </b-field>
+
+      <b-field label="Powtórz hasło">
+          <b-input type="password" v-model="repeatedPassword"
+              password-reveal
+              minlength="8"
+              maxlength="30"
+              required>
+          </b-input>
+      </b-field>
+
+      <div class="notification is-danger" v-if="errors.length">
+        <p v-for="error in errors" v-bind:key="error">{{ error }}</p>
+      </div>
+
+      <br>
+      
+      <button id="btnRegister" class="button">ZAREJESTRUJ SIĘ</button>
+
+      <br>
+      <br>
+
```

Comment:
Jeszcze usunął bym w rejestracji 1~2 ```<br>``` aby nie było scroll w ekranie rejestracji:

![obraz](https://user-images.githubusercontent.com/68618094/201514526-e4665de3-251b-46e0-8df7-d403065d176e.png)

Reply:
Samo usunięcie br nie starczy, trzeba zmienić wielkość komponentów (bo u mnie się renderuje to większe), ale spróbowałem się tym zająć

Reply:
![obraz](https://user-images.githubusercontent.com/80279763/201528263-c1b54637-c077-408a-ad3b-e2c70942640b.png)

Reply:
Jak wyżej widać nawet po ustawieniu dla <b-input> size="is-small" oraz horizontal dla <b-field> to i tak się nie mieści

Reply:
*horizontal dla<b-field> oczywiście

Reply:
Jak widać wyżej nawet po ustawieniu dla <b-field> horizontal oraz dla <b-input> size="is-small" to i tak się nie mieści

Reply:
Ok ważne że jak jest bez błędu to się mieści